cpp/src/parquet/arrow/reader_internal.cc

```diff
@@ -342,21 +342,24 @@ void AttachStatistics(::arrow::ArrayData* data,
           static_cast<::parquet::TypedStatistics<ParquetType>*>(statistics);
       const ArrowCType min = typed_statistics->min();
       const ArrowCType max = typed_statistics->max();
-      if (std::is_floating_point<ArrowCType>::value) {
+      if constexpr (std::is_same<ArrowCType, bool>::value) {
+        array_statistics->min = static_cast<bool>(min);
+        array_statistics->max = static_cast<bool>(max);
```

Comment:
Is min-max useful for boolean? Do they only useful for pruning or agg on same value?

Reply:
Yes. If both of `min` and `max` are `true`, we can assume that the column only has `true` (or `null` if `null_count` is not `0`).